### ISSUE TITLE
UI is off.  Can't click the "Install Web App" button because I cannot scroll down to it.  

### ISSUE THREAD

**Issue by TxTechnician** [filips123/PWAsForFirefox#569]
### Description

This is a fresh install of Opensuse Tumbleweed running Firefox 130.0.1

The UI looks off.  Look at the search bar in this photo, the text input box is cut off slightly:
![image](https://github.com/user-attachments/assets/f138f8c4-37bb-47f8-9a8a-fde4525489f0)

I'm unable to create PWAs because I'm also unable to scroll down to the "Install Web App" button.  I can scroll down to where it will appear.  But when I go to click on it.  The UI scrolls back above the button (workaround is to use the tab key and press enter btw.)

### Steps to Reproduce

1. New install of Opensuse Tumbleweed (has ff 130.0.1).
2. Install extension and observe

### Additional Information

```
03:28:49 [WARN] This will download the unmodified Mozilla Firefox and locally modify it
03:28:49 [WARN] Firefox is licensed under the Mozilla Public License 2.0
03:28:49 [WARN] Firefox is a trademark of the Mozilla Foundation in the U.S. and other countries
03:28:49 [WARN] This project is not affiliated with the Mozilla Foundation in any way
03:28:49 [WARN] By using this project you also agree to the Firefox Privacy Notice: https://www.mozilla.org/privacy/firefox/
03:28:49 [WARN] Check the Firefox website for more details: https://www.mozilla.org/firefox/
04:06:36 [WARN] No required 48x48 icon is provided
04:06:36 [WARN] Generating it from other available icons
04:15:50 [WARN] No required 48x48 icon is provided
04:15:50 [WARN] Generating it from other available icons

```

### Environment

* Operating system:  OpenSUSE Tumbleweed
* System architecture: 64_86
* Desktop environment:  KDE Plasma
* Installation method:  packagecloud using opensuse tutorial
* PWAsForFirefox extension version:  2.12.4
* PWAsForFirefox native version: 2.12.4
* PWAsForFirefox runtime version: 130.0.1
* Firefox version: 130.0.1

(I initially installed the newest runtime (2.12.5) but downgraded when the extension said it was mismatched with addon version (firefox 131 isn't available yet, so I assume that the extension version is tied to the version of firefox.

**Comment by filips123** [comment]
Do you have fractional scaling enabled? If so, this is the same issue as #230.

You can try Ctrl+Scrollwheel while hovering over the extension popup. This should zoom in/out the content, and maybe the button will appear. If it doesn't work, you will unfortunately have to disable fractional scaling to make the button appear, or install PWAs from the command line.

Also, you should install the native program version 2.15.5. The extension hasn't been updated yet, as it's awaiting review, but it should become available soon (hopefully before Firefox 131 is released). The wording in the extension is slightly misleading, so I will try to improve it in the future.

**Comment by TxTechnician** [comment]
> Also, you should install the native program version 2.15.5. 

I do, downgrading was part of the troubleshooting.  And I shouldn't have stopped there.  Because this isn't a problem with your addon.  It's a Wayland issue.  My UI scale is off.  And it is causing other addons to not render correctly.  I'm on X now and everything is fine.

**Comment by TxTechnician** [comment]
I have no idea why.  But here is how I fixed this problem on Opensuse Tumbleweed.  

NOTE:  This is a fresh install and I had only used Wayland up till this point.

I logged out of my Wayland session to see if the same thing happens on X. On X, all of my UI looked different.  Much more streamlined and things all looked.... better.  Firefox too.  

I switched back to Wayland.  And everything looks good.  My UI looks better and things are where they should be now.
